samples/host_exerciser/host_exerciser_cmd.h

```diff
@@ -270,21 +278,24 @@ class host_exerciser_cmd : public test_command
          }
 
 
-
-
         std::cout << "Test Completed" << std::endl;
-        //stop performance counter
-        if (perf->stop() != FPGA_OK) {
-            std::cout << "Failed to stop the fpga perf counter" << std::endl;
-            return -1;
+        if (perf) {
```

Comment:
add perf print and stop into same if statement.
line 283 and 293 in sane if statement,
line 286 remove  return -1 and line 297

tools should compare  Compare buffer even perf fails ?

Reply:
I think you want to stop the perf counters are soon as you detect end-of-test.  Otherwise, the counter values can be skewed, if they are time sensitive.